app/src/main/java/com/waracle/vision/toxicplants/PlantDetector.kt

```diff
@@ -25,6 +31,9 @@ class PlantDetector {
 
     lateinit var probabilityBuffer: TensorBuffer
 
+    private lateinit var predictionLabels: List<String>
+
```

Comment:
There is an extra space here that you can get rid of.
I haven't included the auto-reformat code and lint validations.